src/assets/scripts/client/App.js

```diff
@@ -50,8 +51,55 @@ export default class App {
         this.prop.log = LOG.DEBUG;
         this.prop.loaded = false;
 
-        return this.setupHandlers()
-            .loadInitialAirport(airportLoadList, initialAirportToLoad);
+        $.getJSON('assets/airports/airportLoadList.json')
```

Comment:
This should live in it's own method called from the `constructor`

Reply:
moved to `_fetchAirportLoadList`